### ISSUE TITLE
Object.activate is deprecated.

### ISSUE THREAD

**Issue by fitoflos** [AtomLinter/linter-htmlhint#31]
AtomLinter legacy API has been removed. Please refer to the Linter docs to update and the latest API: https://github.com/atom-community/linter/wiki/Migrating-to-the-new-API
```
Object.activate (/Users/Fitty/.atom/packages/linter/lib/main.coffee:52:6)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:240:19)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:221:30)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15)
Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:213:14)
PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:452:21)
```

**Comment by Shmink** [comment]
+1

**Comment by Stephen304** [comment]
+1

**Comment by salvimateus** [comment]
+1

**Comment by thorade** [comment]
This is a duplicate of #27 which has been closed, 
because it has been fixed in version 0.1.1 of the plugin.
Please confirm and close this issue.
